scripts/start_server.sh

```diff
@@ -7,5 +7,5 @@
 # The directory in which this script resides.
 readonly BASEDIR=$(dirname $BASH_SOURCE)
 
-dev_appserver.py -A cr-status --enable_console=1 \
+dev_appserver.py -A cr-status --enable_console=1 --datastore_emulator_port=15606 \
```

Comment:
Actually, the Cloud Datastore Emulator is now the default datastore implementation of `dev_appserver.py` unless the flag `--support_datastore_emulator` is set to `false`. Hence, when I run this script, the Cloud Datastore Emulator runs **FIRST**, on a **RANDOM** port number followed by the development server. 

Somehow, the old GAE DB Client Library was automatically able to read the port number of the DataStore Emulator. However, the same is not true for Cloud NDB Library. As it says [here](https://cloud.google.com/datastore/docs/tools/datastore-emulator#setting_environment_variables),

> After you start the emulator, you need to set environment variables so that your application connects to the emulator instead of your production Datastore mode database.

If the environment variable `DATASTORE_EMULATOR_HOST` is not set, Cloud NDB will try to access the production server. 

As the port number on which the DataStore Emulator is random and **CHANGES** every time we run the `dev_appserver.py`, I had to set the flag `--datastore_emulator_port=15606` so that I could set the following environment variable -  `DATASTORE_EMULATOR_HOST: 'localhost:15606'`. If we do not use the flag, then we would have to change the environment variable each time we run `dev_appserver.py`

Reply:
Regarding JRE, I had JRE installed on my machine before I installed this repo. Maybe, that's why I never faced any problem while using Datastore emulator.

Reply:
Ah, thanks for explaining that.  So, you decided on the port number and put the same value in two places that are consistent with each other.  It's kind of disappointing that dev_appserver cannot do that itself.

As for the JRE, that makes sense.  We should probably just add a note to the README.md saying that it is needed and including the link to explain why.  It expect that it will be an easy prerequisite to satisfy, even in the CI environment.